packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -61,13 +64,23 @@ export async function runWithRetry<T>(
                 }, err);
                 throw err;
             }
+
+            if (progress.cancel?.aborted === true) {
+                throw new NonRetryableError(
+                    "runWithRetryAborted",
+                    undefined,
+                    DriverErrorType.genericError,
```

Comment:
I'd not use generic error.
This is same as this code:
```ts
        if (error.name === "AbortError") {
            throw new RetryableError(
                "Fetch Timeout (AbortError)", OdspErrorType.fetchTimeout, { driverVersion });
        }
```

And you have https://github.com/microsoft/FluidFramework/issues/8488 that asks to introduce and start using new OdspErrorType.fetchAborted error.

I'd do it here, though it has back-compat considerations, so likely we have to start with OdspErrorType.fetchTimeout here and use this bug to move all usages to new code.
Also I think such codes should not be odsp-specific, so should be added in different enum

Reply:
since it's not odsp specific, we would need to add to `DriverErrorType` in driver-definitions, which also has back-compat considerations, and requires pre-release etc. 
is it ok to use `OdspErrorType.fetchTimeout` here?